libs/postgres_ffi/src/pg_constants.rs

```diff
@@ -11,12 +11,6 @@
 
 use crate::{BLCKSZ, PageHeaderData};
 
-//
-// From pg_tablespace_d.h
-//
-pub const DEFAULTTABLESPACE_OID: u32 = 1663;
-pub const GLOBALTABLESPACE_OID: u32 = 1664;
```

Comment:
Is it worth calling out here that some constants are in `postgres_ffi_types`? Or maybe even re-export them for convenience?

Reply:
Added a note here